docker/data-science/notebooks/ch13/1.setting-up-the-outdoors-dataset.ipynb

```diff
@@ -779,7 +567,7 @@
      "name": "stdout",
      "output_type": "stream",
      "text": [
-      "({'id': '20725', 'url': 'https://outdoors.stackexchange.com/questions/20725', 'title': 'How do people carry camping propane in a car when traveling?'}, {'title': ['How do people carry <em>camping</em> propane in a car when traveling?'], 'body': [\"I'm very new to <em>camping</em>. In fact..I'm trying to prepare for my first <em>camping</em>. One thing that I\"]})\n"
+      "({'id': '20725', 'url': 'https://outdoors.stackexchange.com/questions/20725', 'title': 'How do people carry camping propane in a car when traveling?'}, {'title': ['How do people carry <em>camping</em> propane in a car when traveling?'], 'body': [\"Is it safe to travel in a car with <em>camping</em> propane? What's the <em>safety</em> guideline?\"]})\n"
```

Comment:
What changed with these results (and the ones below)... is it the highlighting? did something change with the highlighting config if so? Just want to verify these changes are intentional (and good).

Reply:
This was from incorrect logic when transforming the generic filter model. The logic was updated but I didn't re-run this notebook. This has been fixed